### ISSUE TITLE
docu: add missing functions to "groups"

### ISSUE THREAD

**Issue by mrustl** [KWB-R/kwb.utils#8]
Running pkgdown::build_site() currently complains that there are missing functions listed in the [_pkgdown.yaml](https://github.com/KWB-R/kwb.utils/blob/7a342f72f0cf6039b204bfc96a25d4262aeff3b7/_pkgdown.yaml)

```
Writing 'reference/index.html'
 Topics missing from index: decode, .defaultIf, .guessSeparator.1, .guessSeparator.2, .isNetworkPath, .log, .logline, .logok, .logstart, .OStype, encode, findPartialDuplicates, isASCII, isDotOrDoubleDot, printable_chars, removeDuplicates, rowOrColumnwisePercentage
```

See: 
http://kwb-r.github.io/kwb.utils/dev/reference/index.html

**Comment by mrustl** [comment]
Pkgdown still complains

https://github.com/KWB-R/kwb.utils/runs/6632021313?check_suite_focus=true

**Comment by mrustl** [comment]
Needs to be fixed to get the docu website (v0.8.0) up to date

**Comment by hsonne** [comment]
Fixed with #36
